package.json

```diff
@@ -175,7 +175,8 @@
     "eslint-config-ckeditor5": "^3.0.0",
     "postcss-loader": "^4.0.0",
     "colors": "1.4.0",
-    "mini-css-extract-plugin": "2.4.2"
+    "mini-css-extract-plugin": "2.4.2",
```

Comment:
This is already fixed, so we can remove the entry (https://github.com/webpack-contrib/mini-css-extract-plugin/releases/tag/v2.5.1).